content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,181 @@
+
+[Kaidan 0.5.0](https://www.kaidan.im/2020/04/06/kaidan-0.5.0/) - Bam! Check out their new onboarding!
+
+Let's sum it up like this: Remove of OTR + Rework conference and contact details + show PDF previews + title for audio files = [Pix-Art Messenger 2.3.5](https://github.com/kriztan/Pix-Art-Messenger/blob/master/CHANGELOG.md)
+
+Four letters for Windows Users: [UWPX 0.25.0](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.25.0.0) is online with Windows10X support (what ever that Windows 10 is) and a push server. So give it a try!
+
+Monal continues the fight on the iOS front: In the upcoming [beta](https://monal.im/blog/4-6-betas/) are support for geo-location messages, [TLS ALPN support](https://github.com/anurodhp/Monal/issues/206) and improvements on [OpenFire push notifications](https://github.com/anurodhp/Monal/issues/354) as well as many fixes with smack.
+
+Wow - no late April joke - Conversations released version 2.8.0 adding [audio and video calls](https://twitter.com/iNPUTmice/status/1254662039592742913)! Insane!
+
+
+### Servers
+
+Guus der Kinderen, from the Ignite Realtime Foundation Board, wrote a short article on [how to setup a STUN and TURN service in Openfire for audio and video calls](https://discourse.igniterealtime.org/t/preparing-openfire-for-audio-video-calls-with-conversations/87828).
+
+
+### Clients and applications
+
+The maintainer of libstrophe, [Pasis](https://github.com/pasis), and contributor to Profanity, created a tool named [xmppconsole](https://github.com/pasis/xmppconsole) which sends raw XMPP stanzas over an XMPP connection and displays the XMPP stream. Main purpose is to study XEPs and debug servers implementation.
+The tool is under development. The finished version will support both GTK UI and ncurses UI. In their blog, they also explained how to easier contribute commits to their repository, [read on](https://profanity-im.github.io/blog/post/contributing-a-patch-via-github/)!
+
+## Services
+
+yax.im updated their servers from prosody 0.11 to [prosody-trunk](https://hg.prosody.im/trunk/file/d0e6d5bc7ea2/CHANGES). Read more about the changes [here](https://yaxim.org/blog/2020/04/23/new-prosody-on-yax-dot-im/). Unfortunately with a first [outage](https://yaxim.org/blog/2020/04/27/multi-day-message-archive-outage/) :(
+
+On April 25th, [observe.jabber.network](https://observe.jabber.network) was launched into public beta. It offers free monitoring-as-a-service for federated XMPP domains, checking connectivity via c2s and s2s, as well as optionally in-band registration and federated XMPP pings. Application details and more information are available on the website.
+
+
+## Extensions and specifications
+
+
+### Deferred
+
```

Comment:
We should either remove the 'Deferred' headline, or add an item '-' to the category.

Reply:
Those are only template placeholders. Will remove those now